lib/kredis/types/list.rb

```diff
@@ -33,6 +41,10 @@ def last(n = nil)
     n ? lrange(-n, -1) : lrange(-1, -1).first
   end
 
+  def expire_in(seconds)
+    expire seconds.to_i
```

Comment:
Don't think we need this. You can just call `expire expires_in if expires_in` inline.

Reply:
I think I borrowed this code from scalar.rb, but it certainly would not have been necessary to define it in a method.